README.rst

```diff
@@ -392,10 +392,76 @@ returned by the server in the specified time. For example:
+                           options={"cached_queries": {"allowed": ["dave_barry_in_cyberspace_sales"]}})
+
+    # Display all access keys associated with this project.
+    keen.list_access_keys()
+
+    # Get details on a particular access key.
+    keen.get_access_key(access_key_id="ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+    # Revoke (disable) an access key.
+    keen.revoke_access_key(access_key_id="ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+    # Unrevoke (re-enable) an access key.
+    keen.unrevoke_access_key(access_key_id="ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+    # Change just the name of an access key.
+    keen.update_access_key_name(access_key_id="ABCDEFGHIJKLMNOPQRSTUVWXYZ", name="Some_New_Name")
+
+    # Add new access key permissions to existing permissions on a given key.
+    # In this case the set of permissions currently contains "writes" and "cached_queries".
+    # This function call keeps the old permissions and adds "queries" to that set.
+    #     ("writes", "cached_queries") + ("queries") = ("writes", "cached_queries", "queries")
+    keen.add_access_key_permissions(access_key_id="ABCDEFGHIJKLMNOPQRSTUVWXYZ", permissions=["queries"])
+
+    # Remove one or more access key permissions from a given key.
+    # In this case the set of permissions currently contains "writes", "cached_queries", and "queries".
+    # This function call will keep the old permissions not explicitly removed here.
+    # So we will remove both "writes" and "queries" from the set, leaving only "cached_queries".
+    #     ("writes", "cached_queries", "queries") - ("writes", "queries") = ("cached_queries")
+    keen.remove_access_key_permissions(access_key_id="ABCDEFGHIJKLMNOPQRSTUVWXYZ", permissions=["writes", "queries"])
+
+    # We can also perform a full update on the permissions, replacing all existing permissions with a new list.
+    # In this case our existing permissions contains only "cached_queries".
+    # We will replace this set with the "writes" permission with this function call.
+    #     ("cached_queries") REPLACE-WITH ("writes") = ("writes")
+    keen.update_access_key_permissions(access_key_id="ABCDEFGHIJKLMNOPQRSTUVWXYZ", permissions=["writes"])
+
+    # Replace all existing key options with this new options object.
+    keen.update_access_key_options(access_key_id="ABCDEFGHIJKLMNOPQRSTUVWXYZ", options={"writes": {
+        "autofill": {
+            "customer": {
+                "id": "93iskds39kd93id",
+                "name": "Ada Corp."
+            }
+        }
+    }})
+
+    # Replace everything but the key ID with what is supplied here.
+    # If a field is not supplied here, it will be set to a blank value.
+    # In this case, no options are supplied, so any options will be removed.
+    keen.update_access_key_full(access_key_id="ABCDEFGHIJKLMNOPQRSTUVWXYZ", name="Strong_Bad", is_active=True, permitted=["queries"])
+
+
+Create Scoped Keys (Deprecated)
```

Comment:
Maybe deprecated in bold to make it more obvious. Another option is to move in Legacy/Deprecated section

Reply:
Good suggestion. For now I'll bold it. Later I'll likely do as you suggest with a dedicated section for such.